packages/commonwealth/server/migrations/20240612153431-added-isWelcomeOnboardFlowComplete-to-users.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    return queryInterface.sequelize.transaction(async (t) => {
+      await queryInterface.addColumn(
+        'Users',
+        'isWelcomeOnboardFlowComplete',
```

Comment:
We should be using snake_case for all field names. SQL standard is case-insensitive which is why we need the quotation marks when we use mixed casing which is a hassle.